uplc-syntax.md

```diff
@@ -43,42 +69,54 @@ module UPLC-CONCRETE-SYNTAX
                        | "equalsByteString"
                        | "lessThanByteString"
                        | "lessThanEqualsByteString"
+```
 
-  /* builtin functions for cryptography */
+### builtin functions for cryptography
 
+```k
   syntax BuiltinName ::= "sha2_256"
                        | "sha3_256"
                        | "blake2b_256"
                        | "verifySignature"
+```
 
-  /* builtin functions for string */
+### builtin functions for string
 
+```k
   syntax BuiltinName ::= "appendString"
                        | "equalsString"
                        | "encodeUtf8"
                        | "decodeUtf8"
+```
 
-  /* builtin functions that use types as parameters */
+### polymorphic builtin functions
 
+```k
   syntax BuiltinName ::= "ifThenElse"
                        | "chooseUnit"
                        | "trace"
+```
 
-  /* builtin functions that operate on Pair */
+### builtin functions that operate on pairs
 
+```k
   syntax BuiltinName ::= "fstPair"
                        | "sndPair"
+```
 
-  /* builtin functions that operate on List */
+### builtin functions that operate on lists
 
+```k
   syntax BuiltinName ::= "chooseList"
                        | "mkCons"
                        | "headList"
                        | "tailList"
                        | "nullList"
+```
 
-  /* builtin functions that operate on Data */
+### builtin functions that operate on data
```

Comment:
These become section headers when rendered as documentation (PDF or HTML), so we should probably capitalize them (basically how you would expect them to appear in a spec itself).